wiki/Skinning/de.md

```diff
@@ -0,0 +1,28 @@
+# Skinning
+
+Das Skinning ist eines der Schlüsselmerkmale in osu!. Es ermöglicht Spielern, von den ursprünglichen Skinelementen abzuweichen, um ihre eigenen zu erstellen! Skins können von spaßeshalber, thematisch zu PRO variieren oder nahezu unmöglich zu spielen sein.
```

Comment:
```suggestion
Das Skinning ist eines der Hauptmerkmale in osu!. Es ermöglicht Spielern, aus den ursprünglichen Skinelementen ihre eigenen zu erstellen! Skins können variieren, seien sie nun spaßeshalber, thematisch, pro oder nahezu unmöglich zu spielen.
```

The last sentence is a weird one. I tried to come up with something, but I'm not sure if it's better than yours.